### PR TITLE
Patch gtec2wav

### PR DESCRIPTION
**Pull request by deep-introspection**
Correct for wobble
Adjust sensor positions
Add Dwingeloo profile correction
Add compatibility with Python 3.6
Remove the dependency with obspy

### CODE REVIEW

File: module/cogito/gtec2wav.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
-# date            : 2017
-# version         : 1
-# usage           : python gtec2wav.py gtecfile.hdf5
+# date            : 2018
+# version         : 2
+# usage           : python gtec2wav.py
 # notes           :
-# python_version  : 2.7
+# python_version  : 2.7, 3.6
 # license         : BSD (3-clause)
```
Comment (robertoostenveld) [inline module/cogito/gtec2wav.py]: @stephenwhitmarsh I believe this is inconsistent with what you had in mind for EEGsynth

**Comment by robertoostenveld** [comment]
this is only for the offline version right? The wobble is present in the online signal...

**Comment by deep-introspection** [comment]
This is indeed only for the offline version, in preparation for the next Cogito event.

To facilitate this conversion step on the event day I made the offline script compatible with both Python 2.7 and 3.6.

**Comment by robertoostenveld** [comment]
@stephenwhitmarsh can you review and merge this?

**Comment by deep-introspection** [comment]
For the wobble, the cause is apparently an aliasing of the dirac encoding the positions of the electrodes. I suggested to lower the scaling variable to 0.1 instead of 0.5 since those values are static and thus will apprear nicely on averages.

**Comment by stephenwhitmarsh** [comment]
Great, Thanks guys!

@robertoostenveld, yes, EEGsynth is expected to run on Python 2.7, but testing scripts on python 3  can't hurt. @deep-introspection - do you use Anaconda for that? However, I don't understand why you say 'to facilitate this conversion step'. We were not planning on running it on python 3.

@deep-introspection Were you planning on implementing it in the realtime (cogito.py) code as well? If you want, I will then make it more consistent with the rest of the code, i.e. using the ini file for setting all parameters, making it independent from the specific current gtec setup, etc, or in any case create enough documentation in the code to do so later. If you do this within a week from now, we can update the cogito setup at my place, that Robert configured to be 'plug and play'  with two raspberry pi's and a router,  and which we will be bringing to ArtTec

Alternatively, we do this together, in a weekend from now, and/or on the train to Rotterdam. In any case, to be able to do that, we need to be able to playback gtec data directly, or first converting it to edf. Any suggestion guys? 

To be clear, we have the opportunity to run the full cogito at ArtTec (sans telescope). It would be a great opportunity to at least have to potential to do that. And I think it would be good to be flexible just in case plans change last minute.

Personally, I think we should try to finish this up now, so we can just enjoy the pleasure of it all working, and explore the rest of the project.

**Pull request merged**